MyLinkedList.java

```diff
@@ -1 +1,147 @@
+package com.company;
 
+public class MyLinkedList<T> implements List<T> {
+
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
+    private StringBuilder stringBuilder = new StringBuilder();
+
+    @Override
+    public void add(T value) {
+        Node<T> newNode = new Node<>(null, value, null);
+        if (isEmpty()) {
+            first = newNode;
+            last = newNode;
+        } else {
+            last.next = newNode;
+            newNode.prev = last;
+            last = newNode;
+        }
+        size++;
+    }
+
+    @Override
+    public void add(T value, int index) {
+        Node<T> newNode = new Node<>(null, value, null);
+        if (index == 0) {
```

Comment:
this part of code will not work correctly (method add() - add elements at the end of the list)

Reply:
Ok. I delete it, 
It isn't necessary